Modules/UDP/README.md

```diff
@@ -0,0 +1,61 @@
+# UDP
+
+[![Swift][swift-badge]][swift-url]
+[![Zewo][zewo-badge]][zewo-url]
+[![Platform][platform-badge]][platform-url]
+[![License][mit-badge]][mit-url]
+[![Slack][slack-badge]][slack-url]
+[![Travis][travis-badge]][travis-url]
+
+## Installation
+
+```swift
+import PackageDescription
+
+let package = Package(
+    dependencies: [
+        .Package(url: "https://github.com/VeniceX/UDP.git", majorVersion: 0, minor: 5)
+    ]
+)
+```
+
+## Basic usage
+
+### Send messages
+
+Send a message containing "Hello world" from `localhost:5050` to `localhost:5051`
+
+```swift
+import UDP
+
+let udpSocket = try? UDPSocket(ip: IP(port: 5050))
```

Comment:
I would just use `try` instead of `try` - makes this cleaner and more understandable.